packages/firestore/src/model/field_index.ts

```diff
@@ -79,6 +79,18 @@ export function fieldIndexGetDirectionalSegments(
   return fieldIndex.fields.filter(s => s.kind !== IndexKind.CONTAINS);
 }
 
+/**
+ * Returns the order of the document key component.
```

Comment:
Returns the order of the document key component for the given index

Reply:
Done